content/guards.md

```diff
@@ -18,13 +18,15 @@ As mentioned, **authorization** is a great use case for Guards because specific
 @@filename(auth.guard)
 import { Injectable, CanActivate, ExecutionContext } from '@nestjs/common';
 import { Observable } from 'rxjs';
+import type { Request } from 'express';
```

Comment:
We intentionally haven't used express in any example since they are supposed to be platform-agnostic (should work for Fastify as well)